07-knitr-R.Rmd

```diff
@@ -11,31 +12,44 @@ source("tools/chunk-options.R")
 
 > ## Learning Objectives {.objectives}
 >
-> * To discover `knitr` and the generation of dynamic reports
+> * Understand the value of `knitr`: Generate dynamic documents that include text, code, and results.
+> * Control basic formatting using markdown syntax.
+> * Be able to create, edit, and compile an .Rmd document including code chunks and inline code.
+
+`knitr` is an R package that allows you to organize your notes, code, and results in a single document. It's a great tool for "literate programming" -- the idea that your code should be readable by humans as well as computers! It also keeps your writing and results together, so if you collect some new data or change how you clean the data, you just have to re-compile the document and you're up to date!
 
-`Knitr` is an R package that makes your code neat, pretty, and shows your notes, code, and output simultaneously in an `html` document. You create these documents in `.Rmd` files. You can write in `LateX` or `md`. 
+You write `knitr` documents in a simple plain text-like format called markdown, which allows you to format text using intuitive notation, so that you can focus on the content you're writing. But you still get a well-formatted document out. In fact, you can turn your plain text (and R code and results) into an html file or, if you have an installation of LaTeX and Pandoc on your machine, a pdf, or even a Word document (if you must!).
 
-knitr extracts R code in the input document (.Rmd), evaluates it and writes the results to the output document (html). There are two types of R code: chunks (code as separate paragraphs) and inline R code.
+To get started, install the `knitr` package and re-start RStudio.
```

Comment:
Why do you have them re-start RStudio? In my experience, the option to open an R Markdown file is present whether or not knitr has been installed. Could you please point to some documentation stating otherwise?